### DISCUSSION TITLE
Prevent "Reused translation" check when difference is just about upper-/lower-case letter in the beginning

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10370] [Q&A]
In this example you see the two English terms `Day(s)` and `day(s)`. In German they are translated similar to `Tag(e)`.
Of course this result in a "Reused translation" check error. Can I prevent this somehow for a specific source strings?

![image](https://github.com/WeblateOrg/weblate/assets/11861496/b15d3ac5-84c6-46a0-afab-2286944d09a5)

See also the original string including screenshots of its use in the GUI.
https://translate.codeberg.org/translate/backintime/common/de/?checksum=392ac4a54a17b8f2

See similar discussion at #10369

**Comment by nijel** [comment]
Yes, you can tick "For all languages" checkbox when dismissing. You can also do that manually by adding `ignore-reused` flag to the source string. The flag can be found in the documentation: https://docs.weblate.org/en/latest/user/checks.html#reused-translation
